.github/workflows/ci.yml

```diff
@@ -239,7 +239,7 @@ jobs:
     - name: install packages
       run: |
         ./alpine.sh apk update
-        ./alpine.sh apk add build-base cmake git python3 py3-pip clang ninja
+        ./alpine.sh apk add build-base cmake git python3 py3-pip clang ninja util-linux
```

Comment:
If we need this here then we likely need it in the release too:

https://github.com/WebAssembly/binaryen/blob/15d4091715b1db8f00b4a0fe2c3fe6e60f7b873e/.github/workflows/create_release.yml#L138